pkg/rpc/client/rpc.go

```diff
@@ -459,7 +459,7 @@ func (c *Client) GetState(stateroot util.Uint256, historicalContractHash util.Ui
 func (c *Client) FindStates(stateroot util.Uint256, historicalContractHash util.Uint160, historicalPrefix []byte,
 	start []byte, maxCount *int) (result.FindStates, error) {
 	var (
-		params = request.NewRawParams(stateroot.StringLE(), historicalContractHash.StringLE(), historicalPrefix, historicalPrefix, start)
+		params = request.NewRawParams(stateroot.StringLE(), historicalContractHash.StringLE(), historicalPrefix, start)
```

Comment:
Wow.

Reply:
But can we make `start` optional? It is in the server implementation. Maybe treat `start == nil` as missing `start`?

Reply:
Yes, we can, but then this behaviour will differ from the C# node. C# node throws an exception if `start == nil` (valid way to pass an empty start is to use `[]byte{}`).

Support nil start?

Reply:
I added several checks to the client side, so now user will get the right answer even with nil `prefix` or `start`. But we still may extend our server handler to accept nil parameters.

Reply:
I think it's OK to handle `nil` this way, it will work server-side and the intention is quite clear.